src/Models/FormBuilderResponse.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace FluxErp\Models;
+
+use FluxErp\Traits\HasPackageFactory;
+use FluxErp\Traits\HasUuid;
+use FluxErp\Traits\SoftDeletes;
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\Relations\BelongsTo;
+use Illuminate\Database\Eloquent\Relations\HasMany;
+
+class FormBuilderResponse extends Model
+{
+    use HasPackageFactory;
+    use HasUuid;
+    use SoftDeletes;
+
+    protected $with = ['form', 'user'];
```

Comment:
sure about always eager loading these? Also put each value in extra line, makes it easier to update

Reply:
removed user but kept the form to minimize requests